aggregator/src/app/app.ts

```diff
@@ -12,10 +12,13 @@ import TxTable from "./TxTable.ts";
 import createQueryClient from "./createQueryClient.ts";
 import Mutex from "../helpers/Mutex.ts";
 import Clock from "../helpers/Clock.ts";
+import getNetworkConfig from "../helpers/networkConfig.ts";
```

Comment:
This file does not exist. Looks like you meant to do a rename here.

Reply:
Fixed, missed in rename.